### PR TITLE
Update changelog for 1.23.2

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # C/C++ for Visual Studio Code Changelog
 
+### Version 1.23.2: December 5, 2024
+### Bug Fixes
+* Include clang-format timeout from 10 seconds to 30 seconds. [#10213](https://github.com/microsoft/vscode-cpptools/issues/10213)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Include->Increase

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+* Fix casing of path in include completion tooltip on Windows. [#12895](https://github.com/microsoft/vscode-cpptools/issues/12895)
+* Fix pattern matching of sections in `.editorConfig` files. [12933](https://github.com/microsoft/vscode-cpptools/issues/12933)
+* Fix handling of relative paths passed to cl.exe `/reference` argument. [#12944](https://github.com/microsoft/vscode-cpptools/issues/12944)
+* Fix a leak of compile command file watchers. [#12946](https://github.com/microsoft/vscode-cpptools/issues/12946)
+* Fix a compile commands fallback logic issue. [#12947](https://github.com/microsoft/vscode-cpptools/issues/12947)
+  * Thank you for the contribution. [@yiftahw](https://github.com/yiftahw) [#12948](https://github.com/microsoft/vscode-cpptools/pull/12948)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Should 12946 have a thank you note too? Or should they be combined into 1 bullet point?

Comment (Colengms) [reply Extension/CHANGELOG.md]: I wasn't sure.  I guess both?

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Yeah, I think both should have a thank you or they should be combined into a single bullet point that references both issues.

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* Fix pattern matching of sections in `.editorConfig` files. [#12933](https://github.com/microsoft/vscode-cpptools/issues/12933)
+* Fix handling of relative paths passed to cl.exe `/reference` argument. [#12944](https://github.com/microsoft/vscode-cpptools/issues/12944)
+* Fix a leak of compile command file watchers. [#12946](https://github.com/microsoft/vscode-cpptools/issues/12946)
+  * Thank you for the contribution. [@yiftahw](https://github.com/yiftahw) [#12948](https://github.com/microsoft/vscode-cpptools/pull/12948)
+* Fix a compile commands fallback logic issue. [#12947](https://github.com/microsoft/vscode-cpptools/issues/12947)
+  * Thank you for the contribution. [@yiftahw](https://github.com/yiftahw) [#12948](https://github.com/microsoft/vscode-cpptools/pull/12948)
+* Fix an issue in which a `didOpen` event was processed before the language client was fully started. [#12954](https://github.com/microsoft/vscode-cpptools/issues/12954)
+* Fix IntelliSense issues related to large header files (>32K) and encodings other than UTF-8.
+
+### Enhancements
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: nit: I think we always list the enhancements before the bug fixes.

File: Extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
 # C/C++ for Visual Studio Code Changelog
 
+### Version 1.23.2: December 5, 2024
+### Enhancements
+* Add handling of `-fno-char8_t` and `-fchar8_t` compiler arguments. [#12968](https://github.com/microsoft/vscode-cpptools/issues/12968)
+* Add support for providing well-known compiler argument information to Copilot Completions. [#12979](https://github.com/microsoft/vscode-cpptools/pull/12979)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: `[#12979]`->`[PR #12979]`.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Same for [#12993]

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
Can you add a bullet point for the fix https://github.com/microsoft/vscode-cpptools/pull/12988 -- just mention it fixed unnecessary cancellation of Copilot context requests.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**